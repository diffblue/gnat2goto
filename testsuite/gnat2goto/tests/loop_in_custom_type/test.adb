procedure Test is
   From : constant := 1;
   To : constant := 5;
   subtype My_Type is Integer range From .. To;
   Iter_Counter : Integer := 0;
begin
   for X in My_Type loop
      pragma Assert (X>4);
      Iter_Counter := Iter_Counter + 1;
   end loop;
   pragma Assert (Iter_Counter = To - From + 1);
end Test;
