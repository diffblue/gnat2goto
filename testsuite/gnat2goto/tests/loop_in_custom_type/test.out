[1] file test.adb line 8 assertion: FAILURE
[2] file test.adb line 11 assertion: SUCCESS
VERIFICATION FAILED
